api/DOMMatrix.json

```diff
@@ -98,10 +98,10 @@
           }
         }
       },
-      "fromFloat32Array": {
+      "a": {
         "__compat": {
-          "spec_url": "https://drafts.fxtf.org/geometry/#dom-dommatrix-fromfloat32array",
-          "description": "<code>fromFloat32Array()</code> static method",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrixReadOnly/a",
```

Comment:
Can you make sure `mdn_url` and `spec_url` don't contain "readonly"?

Reply:
The spec URL must contain the `readonly` segment, but I've removed the MDN URLs since they're 404s anyways.

Reply:
I see, there's just one definition, and it actually says what to do for both 👍